mkosi

```diff
@@ -1911,21 +2037,26 @@ def build_image(args, workspace, run_build_script):
         return None, None, None
 
     tar = None
+    luks_keys = {}
+
+    if args.luks_encryption is not None:
+        luks_keys = prepare_luks(args)
```

Comment:
Hmm, I'd prefer if prepare_luks() would check args.luks_encryption internally as first thing, and simply return `None` (or maybe `{}`) if not set. i.e. i think it would be good if the flow in build_image() would now contain any "if" checks, but that they are all hidden inside the functions...